### DISCUSSION TITLE
":Neorg keybind norg core.dirman.new.note" no longer working

### DISCUSSION THREAD

**Discussion by stephenloggedon** [nvim-neorg/neorg#1550] [Q&A]
I use this command to open new notes from alpha.nvim and it seems like after some changes in the last couple of months it no longer works.

```
[neorg] (11:36:40)
...im/lazy/neorg/lua/neorg/modules/core/neorgcmd/module.lua:240
Error when executing `:Neorg keybind` - such a command does not exist!
```

I am using LazyVim and here is my Neorg config:

```
return {
  "nvim-neorg/neorg",
  lazy = false, -- Disable lazy loading as some `lazy.nvim` distributions set `lazy = true` by default
  version = "*", -- Pin Neorg to the latest stable release
  opts = {
    load = {
      ["core.defaults"] = {},
      ["core.concealer"] = {},
      ["core.journal"] = {
        config = {
          journal_folder = "journal",
        },
      },
      ["core.dirman"] = {
        config = {
          workspaces = {
            notes = "~/notes",
          },
          index = "index.norg",
          default_workspace = "notes",
        },
      },
    },
  },
}
```

**Comment by TogiFerretFerret** [comment]
same issue

**Comment by Anrock** [comment]
I've noticed that it's not a specific keybind but whole `:Neorg keybinds` subcommand is missing. Some time before it was wonky - appearing and disappearing depending on how norg file was opened. Now I can't get it even if `:edit`ing a file directly.

**Comment by ALVAROPING1** [comment]
the keybinds module was refactored a while ago, now it uses `<plug>` mappings instead of `:Neorg keybinds` subcommands. You can read more about it [in the wiki](https://github.com/nvim-neorg/neorg/wiki/User-Keybinds). To trigger new note programmatically, you can also call `require("neorg.modules.core.dirman.module").public.new_note()`
